demos/bert_named_entity_recognition_demo/python/models.lst

```diff
@@ -0,0 +1,3 @@
+# This file can be used with the --list option of the model downloader.
+# The demo should support any BERT calibrated/fine-tuned/distilled/etc on the CONLL dataset
```

Comment:
Does it matter what dataset it's been trained on?

Reply:
yes, different datasets have different tags and may require additional steps to handle (e.g. tokenising spec symbols)